english/webdev-nanodegree/udacity-nanodegree.md

```diff
@@ -0,0 +1 @@
+Smart people will choose to study the FullStack Web Development Nano Degree offered by Udacity as it prepares a student to the latest market needs.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.